### DISCUSSION TITLE
Use main branch as stable release branch

### DISCUSSION THREAD

**Discussion by galenking** [Shopify/dawn#2965] [Ideas]
It would be helpful to understand the branch strategy used by the Dawn team—and why `main` is not the stable release branch.

Am I the only one that finds this frustrating? 

When using Dawn for client projects, what is the best approach to staying updated with the latest release?

**Comment by galenking** [comment]
Does anyone have any further info on how best to structure a repo for client themes based on Dawn? If main isn’t stable, what branch should we use?

**Comment by golden-aries** [comment]
I was exploring main and releases branch history today.
At first, from the history graph, it appeared to me that the changes in the release/13.0.1 branch have not been merged into the main branch and subsequently have not been merged into release/14.0.0 branch.

Then I discovered this commit in the main branch:
Pull in latest release 13.0.1 (#3292)
Commit: c5ba94b19e6ffe804ad0c2ebe1d03257df0f1ba7
Date: Fri Feb 23 2024 09:39:21 GMT-0800 (Pacific Standard Time)

It did not show an edge connecting it to the release/13.0.1 branch.
I guess they are somehow merging stable changes manually, without leaving a trace in the git branch history, which is not the best practice, but there might be a reason for that.

That might mean that main branch is in fact is used as a stable branch, because all release/* branches stems from the main one, as far as I can see.
